static/js/fenced_code.js

```diff
@@ -52,6 +52,19 @@ function wrap_quote(text) {
     return quoted_paragraphs.join('\n\n');
 }
 
+function wrap_tex(tex) {
+    try {
+        return katex.renderToString(tex, {
+            displayMode: true,
+        });
+    } catch (ex) {
+        var errspan = document.createElement('span');
+        errspan.className = 'tex-error';
+        errspan.appendChild(document.createTextNode(tex));
+        return errspan.outerHTML;
```

Comment:
This comes in especially handy, since the node tests were complaining for using `document`. Thanks!